include/dsn/cpp/clientlet.h

```diff
@@ -103,7 +103,7 @@ namespace dsn
         }
 
         template<typename THandler>
-        safe_late_task<THandler>* create_late_task(
+        inline safe_late_task<THandler>* create_late_task(
```

Comment:
This is orthogonal to template - simply want to make it inlined to boost the performance.